website/content/api-docs/secret/databases/redis.mdx

```diff
@@ -0,0 +1,80 @@
+---
+layout: api
+page_title: Redis - Database - Secrets Engines - HTTP API
+description: >-
+  The Redis plugin for Vault's database secrets engine generates [ACL credentials](https://redis.io/docs/manual/security/acl/)
+  credentials to access Redis servers.
+---
+
+# Redis Database Plugin HTTP API
+
+The Redis database plugin is one of the supported plugins for the database
+secrets engine. This plugin generates database credentials dynamically based on
+configured roles for the Redis database.
+
+## Configure Connection
+
+In addition to the parameters defined by the [Database
+Secrets Engine](/api-docs/secret/databases#configure-connection), this plugin
+has a number of parameters to further configure a connection.
+
+| Method | Path                     |
+| :----- | :----------------------- |
+| `POST` | `/database/config/:name` |
+
+### Parameters
+
+- `name` `(string: <required>)` - Specifies the name for this database connection.
+This is specified as part of the URL.
+
+- `plugin_name` `(string: <required>)` – Specifies the name of the plugin to use for this connection.
+
+- `host` `(string: <required>)` – Specifies the host to connect to.
+
+- `port` `(int: <required>)` – Specifies the port number of the connection.
+
+- `username` `(string: <required>)` – Specifies the username for Vault to use.
+
+- `password` `(string: <required>)` – Specifies the password corresponding to
+the given username.
+
+- `allowed_roles` `(list: [])` – Specifies whether to skip verification of the
+server certificate when using TLS.
```

Comment:
Looks like specific databases only document their own parameterts and do not repeat the Database common parameters [documented here](http://localhost:3000/api-docs/secret/databases)

So maybe we should not repeat plugin_name & allowed_roles here.